README.md

```diff
@@ -17,6 +17,8 @@ make
 ```
 
 ### Patch and build AFL Fuzzer
+Run `build.sh` or
```

Comment:
I would prefer to use `build_afl.sh` instead of `build.sh`, since it only builds AFL part.

Reply:
Fixed!